test/taskcreator_test.js

```diff
@@ -11,7 +11,7 @@ suite('TaskCreator', function() {
   helper.setup();
 
   // these tests require real TaskCluster credentials (for the queue insert)
-  if (!helper.haveRealCredentials) {
+  if (!helper.haveRealCredentials || process.env.TASK_ID) {
```

Comment:
Oh, I didn't see that you had added this conditional.. what this is saying is "if we have no credentials or are running as  a task, skip".  That's not what we want :)

Reply:
@djmitche Yes because on the initial conditional (if (!helper.haveRealCredentials)) it does not seem to run the test. I removed it just now.. But the test is still pending (for the taskcreator part). 

I realize that my previous conditional is not correct.

Is it supposed to be `if (!helper.haveRealCredentials && !process.env.TASK_ID)` instead? This way it will only skip if both the credentials and not running as a task. What do you think?